gate-web/gate-web.gradle

```diff
@@ -50,6 +50,9 @@ dependencies {
     exclude group: 'org.springframework.security'
   }
 
+  // IAP
+  compile('com.nimbusds:nimbus-jose-jwt:5.2')
```

Comment:
Is this a well known JWT library? Any chance we could use a Spring and/or Google library?

Reply:
Yes, this is one of the few well known third party java libraries for jwt, according to https://jwt.io/
I haven't seen Spring specific or Google libraries to use as replacement.